### DISCUSSION TITLE
Is it possible to have floating left and right buttons?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [asmyshlyaev177/react-horizontal-scrolling-menu#229] [Q&A]
Great library! Is it possible to have floating left and right buttons?  Something like this

![image](https://user-images.githubusercontent.com/9777763/224287692-266bd040-1fa7-417a-b0c7-58a8c06957c3.png)

**Comment by asmyshlyaev177** [comment]
Yes, you can style buttons with css.
